src/services/urlAssembly.spec.ts

```diff
@@ -293,4 +295,117 @@ describe('static query', () => {
 
     expect(url).toBe('/?foo=foo&simple=ABC')
   })
+})
+
+describe('host params', () => {
+  test.each([
+    ['https://kitbag.dev'],
+    [host('https://kitbag.dev', {})],
+  ])('given simple route with string host and without params, returns route host', (host) => {
+    const [route] = createExternalRoutes([
+      {
+        name: 'simple',
+        path: '/',
+        host,
+        component,
+      },
+    ])
+
+    const url = assembleUrl(route)
+
+    expect(url).toBe('https://kitbag.dev/')
+  })
+
+  test.each([
+    ['https://[?subdomain]kitbag.dev'],
+    [host('https://[?subdomain]kitbag.dev', { subdomain: String })],
+    [host('https://[?subdomain]kitbag.dev', { subdomain: withDefault(String, 'abc') })],
```

Comment:
These having the period missing makes sense. But it also makes me think about devx for this sort of thing. 

I wonder if we could allow users to specify a period or slash is optional without using  param.